src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/SvnContainer/config/supervisord.conf

```diff
@@ -1,11 +1,24 @@
 [supervisord]
 nodaemon=true
+user=root
 
 [program:apache2]
-command=/bin/bash -c "exec /usr/sbin/apache2 -DFOREGROUND"
+command=/bin/bash -c "source /etc/apache2/envvars && /usr/sbin/apache2 -DFOREGROUND"
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
```

Comment:
this tells supervisor to not put the output from the commands into a separate file but use sys.out sys.err (which will then be visible in `docker log`)